CanvasCore/CanvasCore/Extensions/CKIClient+Extensions.swift

```diff
@@ -36,7 +36,7 @@ extension CKIClient {
         let url = baseURL ?? URL(string: "")!
         var masqueradeID: String? = nil
         if let actAsUserID = actAsUserID, actAsUserID.count > 0 { masqueradeID = actAsUserID }
-        let session = Session(baseURL: url, user: currentUser.sessionUser(), token: accessToken, masqueradeAsUserID: masqueradeID)
+        let session = Session(baseURL: url, user: currentUser.sessionUser(), token: accessToken, refreshToken: refreshToken, clientID: clientID, clientSecret: clientSecret, masqueradeAsUserID: masqueradeID)
```

Comment:
I just realized that `authSession` is not getting updated when the token refreshes. I think I will have a way to create a `CanvasCore.Session` from a `Core.LoginSession` and replace all uses of `CKIClient.current?.authSession` with `AppEnvironment.shared.currentSession?.authSession`.